From 440a4cc2475ff9ed1fc7b52b965dded8a36c9a2d Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 18 Feb 2011 13:11:53 +0100
Subject: [PATCH] Makefile: do not dash syntax check module-setup.sh

module-setup.sh is bash syntax, so dash complains about bash contructs
---
 Makefile |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/Makefile b/Makefile
index 9dbe981..ab69c16 100644
--- a/Makefile
+++ b/Makefile
@@ -95,6 +95,7 @@ gitrpm: dracut-$(VERSION)-$(GITVERSION).tar.bz2
 
 check: all
 	@ret=0;for i in modules.d/99base/init modules.d/*/*.sh; do \
+                [ "$${i##*/}" = "module-setup.sh" ] && continue; \
 		dash -n "$$i" ; ret=$$(($$ret+$$?)); \
 	done;exit $$ret
 	$(MAKE) -C test check
